src/link/linkconfiguration.cpp

```diff
@@ -11,6 +14,7 @@ const QMap<LinkConfiguration::Error, QString> LinkConfiguration::_errorMap {
     {InvalidArgsNumber, "Link configuration have a invalid number of arguments"},
     {ArgsAreEmpty, "Link configuration arguments are empty."},
     {InvalidUrl, "Url not formatted properly."},
+    {InvalidSubnet, "IP is not in a reachable network."},
```

Comment:
```suggestion
    {InvalidSubnet, "IP is not in a reachable network. It may not respond to IP change messages. Consider adding your device to this network."},
```

Reply:
This is a generic message, should not be device specific.

Reply:
by "device" I meant computer

Reply:
"It may not respond to IP change messages." is a Ping360 specific thing, but I'll keep the final part.

Reply:
Moved to: "IP is not in a reachable subnet. Configure your computer network settings."